pylint_odoo/misc.py

```diff
@@ -172,6 +167,65 @@ def _check_{NAME_KEY}(self, module_path)
                         node.file = node_file_original
                         node.lineno = node_lineno_original
 
+    def visit_module(self, node):
+        self.wrapper_visit_module(node)
+
+    def add_message(self, *args, **kwargs):
+        valid_odoo_versions = self.linter._all_options[
+            'valid_odoo_versions'].config.valid_odoo_versions
+        version = (self.manifest_dict.get('version')
```

Comment:
What about if version is not defined in the manifest?

Reply:
Done, I changed the order to get the version